src/bci_build/package/389-ds/README.md.j2

```diff
@@ -0,0 +1,67 @@
+# {{ image.pretty_name }} container image
+
+## Description
+
+[389 Directory Server](https://www.port389.org/) is a highly usable, fully featured, reliable and secure LDAP server implementation.
+
+## Usage
+
+By default, the image launches 389 Directory Server with the same configuration that comes with the SUSE Linux Enterprise Server, with one exception, a pre-configured Name Service Switch (NSS) configuration file (`/etc/nsswitch.conf`) in included.
+
+```ShellSession
+$ podman run -it --rm -p 3389:3389 -p 3636:3636 {{ image.pretty_reference }}
+```
+
+## Volumes
+
+The database is stored in the directory `/data`. A new empty database will be created on container startup unless an existing database is already present in `/data`.
+
+To mount a host directory as a volume for your database, run the following command:
+
+```ShellSession
+$ podman run -it --rm -v /my/own/datadir:/data:Z -p 3389:3389 -p 3636:3636 {{ image.pretty_reference }}
+```
+
+## Certificates
+
+By default, the container will use a self-signed CA certificate and a server certificate signed by that CA.
```

Comment:
Will the server create its own CA?

Reply:
No, it does not, and neither has self-signed CAs.

This sentence is wrong for our image, it does have self-signed for official Quay images. I need to fix this, thanks for noticing.

```
INFO: Checking for PEM TLS files ...
INFO: Have /data/tls/server.key -> False
INFO: Have /data/tls/server.crt -> False
INFO: Have /data/tls/ca -> False
INFO: Have /data/config/pwdfile.txt -> True
INFO: Unable to configure TLS from PEM, missing a required file.
```

Reply:
Are you sure? The images should always create the self signed ca ?